src/v/net/conn_quota.cc

```diff
@@ -9,17 +9,22 @@
 
 #include "conn_quota.h"
 
+#include "bytes/oncore.h"
```

Comment:
Looks like this include needs to be in the header

Reply:
Done.